### PR TITLE
[14.0][FIX] shopinvader: Suitable return to custom/create endpoint

### PR DESCRIPTION
**Pull request by cristiano-one**

### CODE REVIEW

File: shopinvader/services/customer.py [hunk 1]
```diff
@@ -193,3 +199,13 @@ def _prepare_create_response(self, binding):
             "role": binding.role,
         }
         return response
+
+    def _valid_before_binding(self, params):
```
Comment (sadamo) [inline shopinvader/services/customer.py]: Maybe method name should be _validate_before_binding

**Review by sadamo — CHANGES_REQUESTED** [review]

File: shopinvader/services/customer.py [hunk 2]
```diff
@@ -193,3 +199,13 @@ def _prepare_create_response(self, binding):
...
+        fields = ["partner_email", "cnpj_cpf"]
+        values = [params.get("email"), params.get("cnpj_cpf")]
```
Comment (lmignon) [inline shopinvader/services/customer.py]: what's 'cnpj_cpf` ????

File: shopinvader/services/customer.py (hunk 1 above)
Comment (cristiano-one) [reply shopinvader/services/customer.py]: True, but I saw this function for another module.

File: shopinvader/services/customer.py (hunk 2 above)
Comment (cristiano-one) [reply shopinvader/services/customer.py]: I removed the checks from this field and transferred it to a more suitable module.

**Pull request closed without merging**